### PR TITLE
Theming cleanup

### PR DESCRIPTION
**Pull request by citizen428**
_Note: keeping this in draft until #15225 has been merged to prevent accidental merges._

## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

This PR needs to be merged after #15225 and performs some cleanup around theme-related code.

## Related Tickets & Documents

#14835

### UI accessibility concerns?

None.

## Added/updated tests?

- [X] Yes

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: this is just a minor code cleanup, the bigger changes have previously been communicated via Changelog posts.

### CODE REVIEW

File: app/views/credits/new.html.erb [hunk 1]
```diff
@@ -157,7 +157,7 @@
     var stripe = Stripe('<%= Settings::General.stripe_publishable_key %>');
     var elements = stripe.elements();
 
-    <% if current_user.decorate.dark_theme? %>
+    <% if current_user.dark_theme? %>
```
Comment (citizen428) [inline app/views/credits/new.html.erb:160]: We removed this method from the decorator, so this check now uses the predicate method defined by the enum option.

**Comment by aitchiss** [comment]
Can we update the themes in the storybook theme switcher here too? It should just be a case of updating this list of themes to match what we have now:

https://github.com/forem/forem/blob/5069cd681a45ff785f1d13291b0590ccd6bfae1c/app/javascript/.storybook/preview.js#L43

**Comment by citizen428** [comment]
@aitchiss Absolutely, I'll do that right away, thanks for pointing it out!

**Review by jeremyf — APPROVED** [review]

**Review by aitchiss — APPROVED** [review]
LGTM 🚀

**Pull request merged**